core/src/main/java/com/linecorp/armeria/common/metric/MoreNamingConventions.java

```diff
@@ -37,6 +37,7 @@
 /**
  * Provides commonly-used {@link NamingConvention}s.
  */
+@Deprecated
```

Comment:
How about also adding `@deprecated Use Micrometer's default {@link NamingConvention}.` in the Javadoc?